tests/bin/prices.test

```diff
@@ -1,7 +1,7 @@
 # Test prices addon
 
 # by default only market prices are reported
-../../bin/hledger-prices -f-
+../../bin/hledger-prices.hs -f-
```

Comment:
Sorry. I thought that I've put this lines initially by accident to speedup tests locally.